R/interpolate-sequence.R

```diff
@@ -17,23 +17,32 @@ ps_interpolate_sequence <- function(x, sequence = "DateTime", value = "Value",
                                     max_gap = 10L,
                                     method = "linear",
                                     step = 0.5) {
-  check_string(sequence)
-  check_string(value)
-  max_gap <- check_count(max_gap, coerce = TRUE)
+  chk_string(sequence)
+  chk_string(value)
+  max_gap <- as.integer(max_gap)
```

Comment:
It's better not to do as.integer() but instead just use `chk_whole_number()`

Reply:
Ignore the coerce = TRUE argument?

Reply:
I guess it might cause consequences down the line so yes use `as.integer()` but do it after you have done `chk_whole_number()`